packages/cli/test/unit/cmds/validator/keymanager/keystoreCache.test.ts

```diff
@@ -12,6 +12,7 @@ import {LocalKeystoreDefinition} from "../../../../../src/cmds/validator/keymana
 const numberOfSigners = 10;
 
 describe("keystoreCache", () => {
+  vi.setConfig({testTimeout: 10000, hookTimeout: 10000});
```

Comment:
This test is failing with timeout very often.

Reply:
There is already a hook timeout, we should probably stick to `vi.setConfig({ ... })` for setting timeout as it is more visible

https://github.com/ChainSafe/lodestar/blob/00dfa63413e9b2a57dbaea70ac7151ef134b0c05/packages/cli/test/unit/cmds/validator/keymanager/keystoreCache.test.ts#L53